doc/faq.rst

```diff
@@ -135,7 +135,7 @@ interested to look at `scikit-learn-contrib
 
 Why are you so selective on what algorithms you include in scikit-learn?
 ------------------------------------------------------------------------
-Code is maintenance cost, and we need to balance the amount of
+Code maintenance cost, and we need to balance the amount of
```

Comment:
There's something wrong with the sentence now. I agree the former isn't ideal either